react-interactive/src/Menue/Page.tsx

```diff
@@ -38,32 +37,37 @@ export interface IProps {
     /** Icon that will show next to your NavLink */
     Icon?: React.ReactNode,
     /** Name of the path or paths thats used for Dynamic Routing with Name being the root of the path*/
-    Paths?: string[]
+    Paths?: string[],
+    /** Pages where this link should remain acitive */
+    OtherActivePages?: string[]
 }
 
-
 const Page: React.FunctionComponent<IProps> = (props) => {
-
     const [hover, setHover] = React.useState<boolean>(false);
     const context = React.useContext(Context)
     const location = useLocation();
 
-    
     const isPathActive = () => {
         // Use Name as the base check and see if the current path starts with this base.
-        return location.pathname.startsWith(`${context.homePath}${props.Name}`);
+        return location.pathname.startsWith(`${context.homePath}${props.Name}`)
+            || isOtherPathActive();
+    }
+
+    const isOtherPathActive = () => {
+        if (props.OtherActivePages == undefined) return false;
+        for (const path of props.OtherActivePages!) {
+            if (location.pathname.includes(path)) return true;
+        }
     }
-    
-    
+
     if (props.RequiredRoles !== undefined && props.RequiredRoles.filter(r => context.userRoles.findIndex(i => i === r) > -1).length === 0)
         return null;
 
     const linkStyle = {
-        color: isPathActive() ? '#007bff' : '#78828d'
-    };    
-    
+        color: isPathActive() == true ? '#007bff' : '#78828d'
```

Comment:
```suggestion
        color: isPathActive()? '#007bff' : '#78828d'
```

Reply:
I am aware that was needed for the `Lint` fix. But the issue was `isPathActive` is of type `() => boolean|void` if you make the other change `IsPathActive` will be `() => boolean` which would fix the `lint` issue